### DISCUSSION TITLE
Issue Template

### DISCUSSION THREAD

**Discussion by marklawlor** [joe-bell/cva#60] [General]
Hey! Sorry for creating the issues. I found it really confusing that the instructions not to create issues for non-bugs were in the Bug Report template. 

Given the choice between "Bug Report" and "Open a blank issue", I think a lot of users like myself are going to choose "create blank issue" when reporting non-bugs.

I think a second template that only instructs users to create a discussion would be helpful - or simply update the description label of the existing template.

**Comment by joe-bell** [comment]
@marklawlor no problem at all! Sorry for the bluntness on my side

There is a large comment at the top of the issue [instructing the use of discussions](https://github.com/joe-bell/cva/blob/main/.github/ISSUE_TEMPLATE/bug_report.md?plain=1#L14). Is this something you missed? If so, how could I make it clearer? Thanks again 🙏

**Reply by marklawlor** [reply]
![Screenshot from 2022-11-13 20-35-49](https://user-images.githubusercontent.com/3946701/201517344-f1730c86-3f65-4e7f-b9c1-80d5018cbf58.png)

You don't see that unless you choose "Bug Report". But if I'm not reporting a bug - why would I click Bug Report?

Hence you never see that message.

A second template eg "Other Issues" that only directs users to discussions would be much clearer

**Reply by joe-bell** [reply]
Good shout, I’ll take that on board — thank you!
